yt_dlp/extractor/elonet.py

```diff
@@ -34,7 +34,26 @@ class ElonetIE(InfoExtractor):
             'title': 'Minulla on tiikeri',
             'description': 'Pienellä pojalla, joka asuu kerrostalossa, on kotieläimenä tiikeri. Se on kuitenkin salaisuus. Kerrostalon räpätäti on Kotilaisen täti, joka on aina vali...',
             'thumbnail': 'https://elonet.finna.fi/Cover/Show?id=kavi.elonet_elokuva_116539&index=0&size=large&source=Solr',
-        }
+            'manifest_stream_number': 5,
+        },
+        'params': {
+            # AssertionError: Expected test_Elonet_116539.mp4 to be at least 9.77KiB, but it's only 840.00B
+            'skip_download': True,
```

Comment:
```suggestion
            'skip_download': 'dash',
```
fragmented streams cannot be download tested